duckduckgo/privacy-essentials-safari
# Name changed to "DuckDuckGo Privacy for Safari"

Task/Issue URL: https://app.asana.com/0/38424471409662/1202229553802448/f

**Description**:
To better distinguish our macOS products, we are renaming DuckDuckGo Privacy Essentials to DuckDuckGo Privacy for Safari.

**Steps to test this PR**:
1. Make sure app name is DuckDuckGo Privacy for Safari
1. Check following UI elements and make sure they contain "DuckDuckGo Privacy for Safari" or "Privacy for Safari" : Main menu, window title, onboarding view, Home view, Share Feedback view
1. Verify unit tests are passing
1. Take a look at README.md and CONTRIBUTING.md
---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
